src/plotting/plotSettings.component.test.tsx

```diff
@@ -85,17 +116,50 @@ describe('Plot Settings component', () => {
   });
 
   it('does not let the user change the X axis scale if time is selected as the X axis', async () => {
+    props.XAxis = 'timestamp';
+    props.XAxisSettings.scale = 'time';
+    createView();
+
+    const radioGroup = screen.getByRole('radiogroup', { name: 'Scale' });
+    const radioButtons = within(radioGroup).getAllByRole('radio');
+    radioButtons.forEach((radioButton) => {
+      expect(radioButton).toBeDisabled();
+    });
+  });
+
+  it('does not let the user change the Y axis scale if time is selected as the Y axis', async () => {
```

Comment:
I'm pretty sure the user should only be able to select time on the x axis and not the y axis - @mattjSTFC can you confirm? If so, we'll just need to filter out `timestamp` from the Y axis selection and remove this sort of logic and testing.

Reply:
I was not aware of this, would be good for Matt to confirm

Reply:
Yes, I am pretty sure Louise is correct but I am just trying to see if we have anything documented to confirm this.

Reply:
Looking at the requirements spreadsheet, in OG51 these comments were made on David's test plan

OG51:

> Facility Staff must have the option to plot a data channel on the x-axis (instead of time)

Test plan:

> Select three channels for plotting, and check without specifying it, that the default y axis parameter is time for each channel.

Comments:

> Need to check scatter plot functionality [SD]. I didn't think the requirement was that the y-axis should default to time, only that it's possible for the x-axis  to not be time. [GP]

This seems to indicate that the Y axis is allowed to be time? Maybe we should leave it and when demoing the plotting functionality we can double check then

Reply:
I'll send Stephen and Victoria an email to clarify

Reply:
@mattjSTFC it seems likely that this PR will be merged before we get a solid answer from them so when you do get a response either comment it here (I'll still get notified even after merge) or email me and either way I'll make any change they suggest

Reply:
Or just create a task in JIRA if it needs changing